test/e2e/signin.ts

```diff
@@ -0,0 +1,115 @@
+import request from 'supertest';
+import server from '../../src/server';
+
+const CLIENT_ID = 'puqOQ6ZT9U8UsfyzuTIwT';
```

Comment:
@GarfDev With the INITIAL_ACCESS_TOKEN you can register a new (dashboard) client. This should be the payload on the POST /reg endpoint: 
```typescript
const res = await http
                .post('/reg')
                .set({ Authorization: `Bearer ${INITIAL_ACCESS_TOKEN}` })
                .send({
                    application_type: 'web',
                    client_name: 'THX Dashboard',
                    grant_types: ['authorization_code'],
                    redirect_uris: ["https://localhost:8082/signin-oidc"],
                    response_types: ["code"],
                    scope: 'openid dashboard user',
                });
```

The response will hold your client_id and client_secret and you can use those in the signin (but also signup) test and you no longer need those constants and the fixture data.